### PR TITLE
NETOBSERV-1355 autocomplete menu button

### PR DESCRIPTION
**Pull request by jpinsonneau**
## Description

Add a button next to `autocompletes` to open / close suggestions.
Also added a scroller when options are more than 8.

[Screencast from 2024-01-23 11-17-14.webm](https://github.com/netobserv/network-observability-console-plugin/assets/91894519/9351f3e2-e79a-4be1-9132-cf1070281c4e)

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [X] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [X] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: web/src/utils/filter-definitions.ts [hunk 1]
```diff
@@ -251,16 +250,16 @@ export const getFilterDefinitions = (
       | undefined = undefined;
 
     if (d.id.includes('namespace')) {
-      getOptions = cap10(getNamespaceOptions);
+      getOptions = getNamespaceOptions;
```
Comment (jotak) [inline web/src/utils/filter-definitions.ts:253]: is `cap10` replaced by something equivalent somewhere else? I couldn't find that
How does it behave with an extremely long list of items?

Comment (jpinsonneau) [reply web/src/utils/filter-definitions.ts:253]: I the video recording above, you can see when I select `namespace` a scrollbar on the right when the input field is cleared

**Review by jotak — APPROVED** [review]

**Comment by jpinsonneau** [comment]
Thanks @jotak !

**Pull request merged**